fortran/README.md

```diff
@@ -0,0 +1,85 @@
+
+## Good default choices
+
+Here are our suggested default choices for build systems and third party libraries:
+
+- Version control: [GitHub](https://github.com/)
+  - de facto standard for open source software
+  - start from our [Fortran template repository](https://github.com/ssciwr/fortran-project-template) for a repository complete with documentation build, tests and GitHub actions
+- Testing framework: 
+  - [pFUnit](https://github.com/Goddard-Fortran-Ecosystem/pFUnit)
+    - unit tests for Fortran
+  - [pytest](https://docs.pytest.org/en/stable/)
+    - use pytest to drive your Fortran code, harnessing the power of the testing framework and at the same time account for the intricacies of Fortran legacy codebases and workflows - see [an example](https://github.com/ssciwr/powr-refactor/blob/main/test/test_colitest.py)
+- Continuous Integration: [GitHub Actions](https://github.com/features/actions)
+  - well integrated with GitHub
+- Documentation: [Doxygen](https://doxygen.nl/manual/index.html)
+  - can harness your comments from the source code
+- Documentation hosting: [readthedocs](https://docs.readthedocs.io/en/stable/) or [MkDocs](https://www.mkdocs.org/), or GitHub pages (see [general recommendations](../general/README.md))
+- Fortran versions/dialects:
+  - Use of Fortran versions from Fortran90 or later is recommended, due to the dynamic memory allocation and deprecation of GOTO constructs
+ 
+
+Our [Fortran Project Template](https://github.com/ssciwr/fortran-project-template) is a simple way to
+start a new Fortran project with all of the above already set up - just click on the green `Use this template` button. You can try it out and compare to learn how to adapt this for your legacy Fortran project.
+
+## Fortran standards
+
+Fortran has different standards according to its version. Some general remarks are summarized here, loosely based on [the Fortran90 documentation](https://www.fortran90.org/src/best-practices.html), [Fortran-lang](https://fortran-lang.org/learn/best_practices/) and own experience:
+- Fortran implementations generally do not make heavy use of more complex objects and namespaces. Therefore, it is advised to use descriptive names, define them within their scope ideally in module files that are passed through `use` statements, keep names consistent in the scope and separate concerns. Historically, the use of memory was costly, so that legacy codebases often reuse variables and (parts of) matrices. Avoid this practice in newer code and refactor legacy code to be more explicit.
+- Use English language for all variable names, comments etc.
+- Use all lowercase letters for all Fortran constructs (`do`, `subroutine`, `integer`, `intent`, ...)
+- Use four spaces for indent.
+- Declare floats as `real(dp)` with `dp` defined in your parameters module.
+- Refactor any `GOTO` constructs that you chance upon.
+- Use `implicit none` to avoid setting of implicit variables including their types.
```

Comment:
I remember wasting a lot of time debugging strange numerical issues in an inherited f77 code before I discovered this one :-) Maybe it could go first / be in bold?